.github/actions/install-happy/action.yaml

```diff
@@ -87,8 +87,13 @@ runs:
         set -o pipefail
         cd $(mktemp -d)
         echo Input argument indicated to use Happy version: $HAPPY_VERSION
-        if [ -z $HAPPY_VERSION ] || [ $HAPPY_VERSION == "latest" ]
+        if [ -z $HAPPY_VERSION ] && [ -e ".happy/version.lock" ]
         then
+          echo "Happy version was not specified, but project has a .happy/version.lock. Getting version from there."
```

Comment:
Two things:
1. I just pushed a fix to the latest version selection that re-writes a lot of this step in JS so I think I caused a lot of merge conflicts for you here
2. I think passing an arg `happy-version-file` would be more flexible and allow using this feature in monorepos where the happy root is not the same as the repo root (this would be similar to how [setup-go](https://github.com/actions/setup-go) has a go-version and a go-version-file arg, [setup-node](https://github.com/actions/setup-node) is designed that way too). Thoughts?

Reply:
Ok, I can add that. I'll figure out how to work it into the JS. Haven't looked at it yet.